mimic/test/test_loadbalancer.py

```diff
@@ -336,10 +360,28 @@ def test_delete_loadbalancer(self):
         self.assertTrue(len(list_lb_response_body['loadBalancers']), 1)
         self.assertTrue(list_lb_response_body['loadBalancers'][0]['id'] == test2_id)
 
-    def test_get_loadbalancer(self):
+    def test_get_loadbalancer_no_nodes(self):
         """
         Test to verify :func:`get_load_balancers` with on
-        ``GET /v1.0/<tenant_id>/loadbalancers\<loadbalancer_id``
+        ``GET /v1.0/<tenant_id>/loadbalancers\<loadbalancer_id>``.  If there
+        are no nodes on the load balancer, "nodes" but not "nodeCount"
+        appears in the response.
+        """
+        lb_id = self._create_loadbalancer(
+            nodes=[{"address": "1.2.3.4", "port": 80, "condition": "ENABLED"}])
+        resp, body = self.successResultOf(json_request(
+            self, self.root, "GET", self.uri + '/loadbalancers/' + str(lb_id)))
+        self.assertEqual(resp.code, 200)
+        self.assertEqual(body['loadBalancer']['id'], lb_id)
+        self.assertNotIn('nodeCount', body['loadBalancer'])
+        self.assertEqual(len(body['loadBalancer']['nodes']), 1)
+
+    def test_get_loadbalancer_with_nodes(self):
+        """
+        Test to verify :func:`get_load_balancers` with on
```

Comment:
OK, so, this was the way the docstring started already, but you repeated it.

First off (hi @jml!) we should phrase test docstrings like this.  https://plus.google.com/+JonathanLange/posts/YA3ThKWhSAj 

Second, what is even a grammar? :)  "Test to verify get load balancers with on GET (path)"